scripts/devservices-healthcheck.sh

```diff
@@ -20,10 +20,23 @@ delay=5
 
 sleep "$start"
 while (( $try <= $to )); do
+    HEALTHY=1
```

Comment:
nit: Other variables in this script are lowercase, so I would stick to that convention.

Reply:
this part is gone now